depthai_examples/common_src/CameraControl.cpp

```diff
@@ -71,15 +71,21 @@ void CameraControl::setFocus() {
     controlQueue->send(ctrl);
 }
 
+int clamp(int v, int min, int max) {
+    if (v < min) return min;
+    if (v > max) return max;
+    return v;
+}
+
 req_type CameraControl::setRgbExposureRequest(exp_req_msg request, exp_rep_msg response) {
     rgb.auto_exposure = req_get(auto_exposure);
     rgb.region.at(0) = req_get(exposure_x);
     rgb.region.at(1) = req_get(exposure_y);
     rgb.region.at(2) = req_get(exposure_width);
     rgb.region.at(3) = req_get(exposure_height);
     rgb.compensation = req_get(compensation);
-    rgb.time_us = req_get(exposure_time_us);
-    rgb.sensitivity_iso = req_get(sensitivity_iso);
+    rgb.time_us = clamp(req_get(exposure_time_us), 1, 33000);
+    rgb.sensitivity_iso = clamp(req_get(exposure_time_us), 1, 33000);
```

Comment:
@alex-luxonis Is this the same for all modules or can we fetch this from device and set the limits based on the CCM ?

Reply:
I'll investigate